lib/util/pretty-time.ts

```diff
@@ -40,3 +41,38 @@ export function toMs(time: string): number | null {
     return null;
   }
 }
+
+const rangeRegex = regEx(/^(?<operator>(>=|<=|<|>))\s*(?<age>.*)/);
+
+export function satisfiesDateRange(
+  date: string,
+  range: string
+): boolean | null {
+  const grps = range.trim().match(rangeRegex)?.groups;
+  if (!grps) {
+    return null;
+  }
+
+  const { operator, age } = grps;
+  const dateMs = DateTime.fromISO(date).toMillis();
```

Comment:
What happens if the releaseTimestamp string is corrupted and can't be converted to a date?

Reply:
Luxon returns Nan and the we return `null`. Its in the tests. 
Should it be logged? I thought invalid dates will be easy to recognize in config so didn't add one.

Reply:
So if a release time stamp is invalid, does it mean it continues without error, and does not match?

Reply:
Yes